src/interfaces/userInterface.ts

```diff
@@ -0,0 +1,13 @@
+export interface UserProps {
```

Comment:
I'm sorry to comment on this again, but... Could we leave the name of it as "User"? Because UserProps seems like an interface you are creating for a React component, but in this case, it is an entity, a set of data with its own structure. For example, I saw you creating CardProps, AvatarProps, and they reflect the properties you expect to receive in a React Component. But this one is not going to be a Props to a component...
Did I make myself clear? If not, we can discuss more about that!

Reply:
ok then, i changed to User only